src/test/resources/pages/adminEmailCompose.html

```diff
@@ -27,7 +27,7 @@
          </script>
          <script src="/tinymce/js/tinymce/tinymce.min.js" type="text/javascript">
          </script>
-         <link href="http://localhost:8888/tinymce/js/tinymce/skins/lightgray/skin.min.css" rel="stylesheet"/>
```

Comment:
I have to disagree here. This missing replacement is possible if `app.url` does not match `TestProperties.inst().TEAMMATES_URL` because the tinymce link starts with the latter URL.
I'm ready to be proven wrong, but I think the only feasible solution here is for both URLs to be the same whenever God mode needs to be run. The explicit localhost:8888 replacement in God mode is certainly not needed.